src/test/java/org/fundacionjala/coding/manuel/movies/MovieTest.java

```diff
@@ -0,0 +1,94 @@
+package org.fundacionjala.coding.manuel.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public class MovieTest {
```

Comment:
In fact you should not test Movie abstract class
You should test the child classes. I mean NewRelease, Regular, and Children
please split this unit test class.